src/services/auth/plugins/password/schemas.ts

```diff
@@ -98,3 +118,18 @@ export const patchResetPasswordRequest: FastifySchema = {
     [StatusCodes.BAD_REQUEST]: {},
   },
 };
+
+export const getMembersCurrentPasswordStatus: FastifySchema = {
+  tags: ['password', 'current'],
+  summary: 'Get the current password status of the authenticated member',
+  description:
+    'Get the current password status of the authenticated member. By example, if the member has a password defined or not.',
```

Comment:
Might be too simple, change if you think it fits the description.
```suggestion
    'Return whether the authenticated member has a password defined.',
```